### ISSUE TITLE
.cdb file failing to open an no indication as to why

### ISSUE THREAD

**Issue by kkukshtel** [ncannasse/castle#76]
Was working in a totally fine file and then I deleted a column or something and now when I try to open the file it won't open. It clearly does something to the editor though as I can't switch between sheets or anything once I try to load the file. Having some sort of debug output here would be great.

**Comment by kkukshtel** [comment]
Actually zooming out it looks like the basic file loaded when Castle is opened is no longer navigate-able. Can't make new sheets/rows/columns, etc.

Also tried to rebuild from source on this repo and am getting:
```
src/JqPages.hx:50: characters 21-32 : Type not found : vdom.Server
```

**Comment by ncannasse** [comment]
Yes, there was some issue about column deletion where it was not properly removed everywhere.
You need to edit manually your .cdb file using text editor and remove the object that has `"sheetname@columnName"` that was removed.

**Comment by kkukshtel** [comment]
Okay I just cleared out all List fields and it seems to load properly now. Do you know what directly causes it so I can avoid doing it again? Also, is this issue fixed in the 1.6 release?
